packages/clerk-js/src/ui/elements/Tabs.tsx

```diff
@@ -118,8 +118,9 @@ export const Tab = (props: TabProps) => {
     >
       <Button
         onClick={onClick}
-        focusRing={isFocused}
+        focusRing={isActive}
```

Comment:
@desiprisg ❓ isn't this going to have a focused state when activation has occurred with a mouse event as well? Do we want that?

Reply:
Yes it will, however that's not the case in our use case right now since we have fields with `autofocus` below. Top priority is to keep the focus on the tabs when keyboard navigating and allow other elements to take focus when clicking. Did not want to make a big refactor right now, just fix this one issue, but feel free to share any ideas. We should definitely rethink this.